src/langs/python/python-previewer.py

```diff
@@ -216,11 +216,12 @@ def set_window(self, the_window: Gtk.Window):
         if self.window is not None:
             self.window.destroy()
         self.window = the_window
+        # Make sure the preview can be re-opened by using the "Show Preview Window" button.
+        self.window.set_hide_on_close(True)
```

Comment:
Here too I guess. Always use the property when its available.

Reply:
With PyGObject it seems to be preferred to use the methods, probably because it's ambiguous if you are setting a Python or GObject property. But I'll update the Vala code!